templates/keylime.conf.j2

```diff
@@ -0,0 +1,663 @@
+#=============================================================================
```

Comment:
The file header should be
```
{{ ansible_managed | comment }}
{{ "system_role:keylime_server" | comment(prefix="", postfix="") }}
```
and we'll need to add a test for this like https://github.com/linux-system-roles/nbde_server/pull/95

Reply:
Fixed the headers, but did not amend the test yet.